pkg/bootstrapper/bootstrapper.go

```diff
@@ -124,6 +124,8 @@ type winNodeBootstrapper struct {
 	svcMgr *mgr.Mgr
 	// installDir is the directory the the kubelet service will be installed
 	installDir string
+	// logDir is the directory that captures log outputs
```

Comment:
logs of what? Just the wmcb or the components initialized by wmcb?

Reply:
According to https://issues.redhat.com/browse/WINC-155, this directory should capture all logs needed to be made available in Artifacts directory. IMO this is not just limited to kubelet and hybrid-overlay logs but kube-proxy logs as well.
I can update the comment here to reflect that this directory is to be made available in `Artifacts`, if that would help